dmriprep/conftest.py

```diff
@@ -0,0 +1,21 @@
+"""py.test configuration."""
```

Comment:
This file is just boilerplate for pytest to automate things, could be outside your PR (actually I think I'm going to add this to master before your PR).

Reply:
I like it. Reading about pytest fixtures now-- I've never used them before.